### ISSUE TITLE
Typescript module error

### ISSUE THREAD

**Issue by sadiqawos** [honeybadger-io/honeybadger-js#604]
## What are the steps to reproduce this issue?
1. Set up a new Node project using TypeScript (I'm using electron)
2. Add @honeybadger-io/js to the project

## What happens?
Typescript compiler complains about the following
```
node_modules/@honeybadger-io/js/honeybadger.d.ts:3:49 - error TS2307: Cannot find module 'express' or its corresponding type declarations.

3 import { NextFunction, Request, Response } from 'express'
```

## What were you expecting to happen?
No complaints by typescript compiler about `express` since I'm not using it in my project

## Any logs, error output, etc?
```
> tsc && electron dist/app.js --development

node_modules/@honeybadger-io/js/honeybadger.d.ts:3:49 - error TS2307: Cannot find module 'express' or its corresponding type declarations.

3 import { NextFunction, Request, Response } from 'express'
                                                  ~~~~~~~~~
Found 1 error.

```

## What versions are you using?
**Operating System:* Ubuntu 
**Package Version:** 3.2.1
**Browser Version:** N/A

**Comment by joshuap** [comment]
Thanks for the bug report!

The issue may be that express is in our `devDependencies`, which aren't installed. I don't want to add express to our main dependencies just to get the type definitions, so we'll have to come up with some different type definitions (or a workaround).

https://github.com/honeybadger-io/honeybadger-js/blob/master/honeybadger.d.ts#L3

**Comment by sadiqawos** [comment]
Thanks for the prompt response.

If the concern is to keep dependencies to a minimum, why don't you split up the typedef into its own package?

I'm not too familiar with this myself, so I had to read [this](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies) very quickly!

**Comment by joshuap** [comment]
Hi @sadiqawos, this should be fixed in v3.2.2 (just released). Can you try that out and let me know if you have any more problems? Thanks!
